src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+      
+        <legend>Personal information</legend>
+        
+        <div class="input">
+          <label for="surname">Surname:</label>
+          <input 
+          type="text" 
+          id="surname" 
+          name="surname" 
+          autocomplete="off"
+          >
```

Comment:
Check and apply code style [recommendations](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)(point 3)
Apply all over the page

```suggestion
          <input 
            type="text" 
            id="surname" 
            name="surname" 
            autocomplete="off"
          >
```